### ISSUE TITLE
[Feature]  Ability to see notes in scheduled transactions

### ISSUE THREAD

**Issue by bchooxg** [actualbudget/actual#2976]
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

When there are a bunch of scheduled transactions in an account the notes column remains empty, even if they have been populated in the rules section. 
Which makes it quite hard to identify what exactly this transaction is

<img width="1584" alt="image" src="https://github.com/actualbudget/actual/assets/56596358/daee28cf-0569-449a-a486-cff3014ba862">

### Describe your ideal solution to this problem

Quickfix: To query the rules for that specific transaction and get the value of the note, if any. 
More complete fix: Redesign the schedules pages to be able to accept the column value for "notes" 

### Teaching and learning

Need to update the documentation for schedules page to include this additional column

**Comment by Kidglove57** [comment]
I have lots of notes that are entered automatically by my scheduled transactions. After saving the schedule I go back into it and click “edit as a rule”. This then enables me to create a rule for that schedule. I then add a category and a note to the transaction. 

@bchooxg Were you thinking of something more than this?

**Comment by jfdoming** [comment]
> I have lots of notes that are entered automatically by my scheduled transactions. After saving the schedule I go back into it and click “edit as a rule”. This then enables me to create a rule for that schedule. I then add a category and a note to the transaction.
> 
> @bchooxg Were you thinking of something more than this?

I think the notes themselves don't display right now even if set. But I have an open PR to fix that!

**Comment by Kidglove57** [comment]
Agreed! They don’t display in the schedule list until posting has occurred. I felt I might have misunderstood the post but could not be completely sure. My apologies.

**Comment by bchooxg** [comment]
> Agreed! They don’t display in the schedule list until posting has occurred. I felt I might have misunderstood the post but could not be completely sure. My apologies.

My apologies if the post is unclear, I have added more screenshots to make it clearer hopefully

**Comment by matt-fidd** [comment]
This now looks to have been implemented
